wikimedia/wikipedia-ios
# Remove cutoff date for article stack restoration

**Phabricator:** https://phabricator.wikimedia.org/T323473

### Notes
Previously, if the app was booted from memory, it only restored the article navigation stack if it was relaunched before 5am the next day. This PR removes that cutoff date so the app always tries to restore the article stack if booted from memory (while respecting and ceding to the Open app on Search tab setting).

Note that this PR doesn't change anything about _what_ is restored and is only relevant when the app is forced out of memory. When the app remains in memory, returning to the app from background will always return you to where you were.

### Test Steps
1. Open the app and navigate to an article (or navigate through a stack of multiple articles)
2. Background and force quit the app
3. Confirm when you relaunch that the article you were viewing is presented
4. In Settings > Search, toggle on "Open app on Search tab"
5. Background and force quit the app
6. Confirm that relaunching opens the app to the search tab and not the article you were previously browsing
